micro_mg3_0.F90

```diff
@@ -2637,22 +2688,22 @@ subroutine micro_mg_tend ( &
                 + (psacwg(i,k)+pgsacw(i,k))*lcldm(i,k)
            qstend(i,k) = qstend(i,k)+ &
                 (prai(i,k)+prci(i,k))*icldm(i,k)+(psacws(i,k)+bergs(i,k))*lcldm(i,k)+(prds(i,k)+ &
-                pracs(i,k)-psacr(i,k))*precip_frac(i,k)
+                pracs(i,k)-psacr(i,k))*precip_frac(i,k)+vap_deps(i,k)
```

Comment:
I'll move my comment here. My gut says that vap_deps needs a precip_frac multiplier, but I'd love to hear why that's not the case. Is it included in the calculation of the rate already?

Reply:
I see that the rate is multiplied by precip_frac in the calculation in pumas_utils.F90. You can just mark this conversation resolved.